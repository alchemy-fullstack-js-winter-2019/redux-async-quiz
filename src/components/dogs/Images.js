import React from 'react';
import PropTypes from 'prop-types';


export default function DogImages({ image }) {
  return (
    <>
    <h2>DOGGOS</h2>
    <img src={image} />
    </>
  );
}

DogImages.propTypes = {
  image: PropTypes.string.isRequired
};

