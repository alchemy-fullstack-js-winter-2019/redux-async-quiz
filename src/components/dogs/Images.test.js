import React from 'react';
import renderer from 'react-test-renderer';
import Images from './Images';
describe('Images Test', () => {
  it('Renders image', () => {

    const tree = renderer.create(
      <Images /> 
    );
    expect(tree).toMatchSnapshot();
  });
});
