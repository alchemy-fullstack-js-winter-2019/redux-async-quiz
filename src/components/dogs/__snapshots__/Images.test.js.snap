// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Images Test Renders image 1`] = `
Array [
  <h2>
    DOGGOS
  </h2>,
  <img />,
]
`;

exports[`Images Test images 1`] = `
Array [
  <h2>
    DOGGOS
  </h2>,
  <img />,
]
`;
